documentation/onboard-form-template.md

```diff
@@ -0,0 +1,34 @@
+# Teacher services infra - Service onboarding form
+- Service name? e.g. calculate-teacher-pay
+- Domains  ? e.g. school-placements-production.teacherservices.cloud
+- Service short name? e.g. ctp
+- Repository? e.g. https://github.com/DFE-Digital/itt-mentor-services
+- Environments? e.g. develoment, staging, production
+- Review apps? Yes or No
+- Cost centre? e.g. 10168
+- Activity code? e.g
+    - placements: 101423
+    - track: 101056
+- List of developers who will need access to Azure? Do they use a DfE laptop or BYOD? e.g
+    - James Beagle - BOYD
+    - Clinton Morris - DfE laptop
+- GDS assessed? Yes or No
+
```

Comment:
Timeline of the different phases: discovery, alpha, private beta, public beta, live ?
https://www.gov.uk/service-manual/agile-delivery/how-the-discovery-phase-works